core/modules/wsched/SchedulerBase.h

```diff
@@ -105,6 +122,17 @@ class SchedulerBase : public wcontrol::Scheduler {
     int _priorityNext; ///< Priority to use starting with the next chunk.
 
     std::atomic<int> _inFlight{0}; //< Number of Tasks running.
+
+private:
+    /// The true purpose of _userQuerycount is to track how many different UserQuery's are on the queue.
+    /// Number of Tasks for each UserQuery in the queue.
+    std::map<uint64_t, int> _userQueryCounts;
```

Comment:
Agreed. Any thoughts on where it should be defined? It shows up on both the czar and the worker and it would be good to define it once for both. There's QueryId in qmeta/types.h, but including that in worker coded causes issues.